requirements.txt

```diff
@@ -17,3 +17,4 @@ pydot>=1.0.28,<=1.0.29
 psutil>=1.2.1,<=3.4.2
 matplotlib>=1.3.1,<=1.5.1
 scikit-fmm>=0.0.9
+python-magic>=0.4
```

Comment:
It looks like the Ubuntu packaged version is 0.2 (https://packages.ubuntu.com/xenial/all/python-magic/filelist) - despite the fact that the package says it's `5.25`. Can we set the requirement to `==0.2`?